plugins/osdn/common.go

```diff
@@ -166,26 +171,11 @@ func (oc *OvsController) validateNetworkConfig(clusterNetworkCIDR, serviceNetwor
 }
 
 func (oc *OvsController) StartMaster(clusterNetworkCIDR string, clusterBitsPerSubnet uint, serviceNetworkCIDR string) error {
-	// Any mismatch in cluster/service network is handled by WriteNetworkConfig
-	// For any new cluster/service network, ensure existing node subnets belong
-	// to the given cluster network and service IPs belong to the given service network
-	if _, err := oc.Registry.GetClusterNetworkCIDR(); err != nil {
-		subrange := make([]string, 0)
-		subnets, _, err := oc.Registry.GetSubnets()
-		if err != nil {
-			log.Errorf("Error in initializing/fetching subnets: %v", err)
-			return err
-		}
-		for _, sub := range subnets {
-			subrange = append(subrange, sub.SubnetCIDR)
-		}
-
-		err = oc.validateNetworkConfig(clusterNetworkCIDR, serviceNetworkCIDR, subrange)
-		if err != nil {
-			return err
-		}
+	if err := oc.validateNetworkConfig(clusterNetworkCIDR, serviceNetworkCIDR); err != nil {
+		return err
 	}
 
+	// Any mismatch in cluster/service network is handled by WriteNetworkConfig
```

Comment:
It's not really clear what this comment means... You can probably just delete it.